internal/xsql/valuer.go

```diff
@@ -462,7 +462,21 @@ func (v *ValuerEval) Eval(expr ast.Expr) interface{} {
 			return nil
 		}
 	case *ast.Wildcard:
-		val, _ := v.Valuer.Value("*", "")
+		all, _ := v.Valuer.Value("*", "")
+		val, ok := all.(Message)
+		if !ok {
+			return fmt.Errorf("unexpected wildcard value %v", all)
+		}
+		for _, key := range expr.Except {
+			delete(val, key)
```

Comment:
Will this affect the original map which is supposed to be readonly?

Reply:
Yes, it has the potential to affect message. I will fix it.